x-pack/plugins/features/common/feature_kibana_privileges.ts

```diff
@@ -104,6 +104,34 @@ export interface FeatureKibanaPrivileges {
      */
     read?: readonly string[];
   };
+
+  /**
+   * Solutions should specify owners of alerts here which will provide the solution read / write access to those alerts.
+   */
+  rac?: {
```

Comment:
The name `rac` suggests this controls access to Rules and Cases as well as Alerts.
I know this isn't true for Rules (as that's controlled by the `alerting` RBAC).
Not sure about Cases.

So, should this be changed to something more specific? 🤔

Reply:
Feature IDs and sub-feature privilege ids are the only entities that cannot be renamed, as these get translated into Elasticsearch Application Privileges, which are then directly assigned to roles. I thought your features were named `actions` and `stackAlerts`, unless I'm thinking of something else?

Reply:
naming is hard, what about just `SolutionAlerts`? I do agree with you @gmmorris

Reply:
> I thought your features were named actions and stackAlerts, unless I'm thinking of something else?

Ah, sorry, I'm referring to the key in the privileges object, rather than the feature IDs.
We use `alerting`: https://github.com/elastic/kibana/blob/3bc2952216f905620afe019af4b3785e385f000d/x-pack/plugins/stack_alerts/server/feature.ts#L33

@XavierM I don't think `SolutionAlerts` is accurate as it'll be used by Stack Rules too.
Am I right that this will define the RBAC for the alerts as data operations? Feels like this should probably be `alertsAsData` or `alerts`, and we should change ours to be `rules` 🤔 

Though... it raises the question: shouldn't the `alerts` privileges be inferred from `rules` privileges? Otherwise a user can create a rule but can't read the alerts produced by that rule 😬

Reply:
@mikecote I think we might need to do another terminology change on our end - change our privileges key to `rules` instead of `alerting` 🤔

Reply:
> Though... it raises the question: shouldn't the alerts privileges be inferred from rules privileges? Otherwise a user can create a rule but can't read the alerts produced by that rule 😬

We probably have a good reason to have different access controls for alerts, but IMHO inheriting those privileges from the rules make a lot of sense, and is the simplest model I can think of.

Reply:
@dgieselaar for t1 analysts, they're able to update alerts (status) but not able to `write` on rules.

@gmmorris I think the renaming to `rules` would be super helpful